### DISCUSSION TITLE
Add compatibility with Start 11

### DISCUSSION THREAD

**Discussion by user858753257** [valinet/ExplorerPatcher#626] [Ideas]
Start 11 is a similar tool to backport the windows 10 start menu layout into windows 11.

https://www.stardock.com/products/start11/

My guess is that both Programms want to set some settings in the same way at the same time and so the context menu from the Taskbar can't be displayed .

![image](https://user-images.githubusercontent.com/8161283/148044970-726cc5e2-d42d-450a-b4eb-cd7aebc31c11.jpeg)

**Comment by valinet** [comment]
Or Start11 could implement compatibility with EP, seeing how it messes up with stuff it shouldn’t really mess with. The Windows 10 taskbar’s context menu is fine, it doesn’t need “fixing” from Start11.

So, I guess this doesn’t happen with Start11 alone, or EP alone, right? How does the menu look with Start11 alone?
